docs/ref/models/fields.txt

```diff
@@ -700,6 +700,15 @@ or :class:`~django.forms.NullBooleanSelect` if :attr:`null=True <Field.null>`.
 The default value of ``BooleanField`` is ``None`` when :attr:`Field.default`
 isn't defined.
 
+``CompositePrimaryKey``
+-----------------------
+
+.. class:: CompositePrimaryKey(*field_names, **options)
+
+A virtual field used for defining a composite primary key.
+
+See :doc:`/howto/composite-primary-key` for usage details.
+
```

Comment:
We will need more reference detail here. We can't expect a user to read through a how-to when they're looking for a particular detail.

Reply:
I added some more docs, please check!